docs/api-reference/json.rst

```diff
@@ -11,7 +11,9 @@ Project
     Returns metadata (info) about an individual project at the latest version,
     a list of all releases for that project, and project URLs. Releases include
     the release name, URL, and MD5 and SHA256 hash digests, and are keyed by
-    the release version string.
+    the release version string. Metadata returned comes from the values provided
+    at upload time and does not necessarily match the content of the uploaded
```

Comment:
Do you want to add that it's the first upload that sets the value in case multiple distributions are uploaded ?

Reply:
I didn't do that, because I wasn't sure if it was true 🙂 If you're confirming that this is the case, then I can add that.

(Personally, "doesn't necessarily match the content of the uploaded files" is the deal-breaker for my use case, so I don't have a personal need to be more precise than that, but I do want whatever goes here to be accurate, obviously).

Reply:
It's (unfortunately) true, at least until https://github.com/pypa/warehouse/issues/8090 is resolved.

Reply:
Done.